packages/cms-base/components/SwProductAddToCart.vue

```diff
@@ -33,9 +49,16 @@ const { addToCart, quantity } = useAddToCart(product);
 
 const addToCartProxy = async () => {
   await addToCart();
-  pushSuccess(
-    `${props.product?.translated.name} ${translations.product.addedToCart}`,
-  );
+  const errors = getErrorsCodes();
+  errors?.forEach((element) => {
+    const { messageKey, params } = useCartErrorParamsResolver(element);
```

Comment:
you should not use composable inside any function
somthing like:

```ts
const { resolveErrorParam } = useCartErrors()
```

and here
```ts
 errors?.forEach((element) => {
    const { messageKey, params } = resolveErrorParam(element);
    pushError(cmsT(translations.errors[messageKey], params));
  });
```

Reply:
done